res/qml/Mixxx/Controls/Knob.qml

```diff
@@ -67,8 +67,9 @@ Item {
         // by enabling multisampling, so we use 4xMSAA here.
         //
         // See https://www.qt.io/blog/2017/07/07/let-there-be-shapes for details.
-        layer.enabled: true
-        layer.samples: 4
+        property int multiSamplingLevel: Mixxx.Config.getMultiSamplingLevel()
+        layer.enabled: multiSamplingLevel != 0
```

Comment:
technically doesn't make a difference, but use strict comparision for good practice.
```suggestion
        layer.enabled: multiSamplingLevel !== 0
```

Reply:
Is 1x multisampling even a thing? Should this be `multisampling > 1`?